inventory-web-app/constants/sidebar.js

```diff
@@ -0,0 +1,30 @@
+import {
+  HomeOutlined,
+  TableOutlined,
+  InteractionOutlined,
+  UserOutlined,
+  CheckSquareOutlined,
+  ToolOutlined,
+  CreditCardOutlined,
+} from '@ant-design/icons';
+
+// Maps what items each role can see
+export const ROLES_TO_VISIBLE_SIDEMENU_ITEMS_MAP = {
+  admin: ['home', 'inventory', 'transactions', 'customers', 'orders', 'repairs', 'audit-center'],
+  manager: ['home', 'inventory', 'transactions', 'customers', 'orders', 'repairs', 'audit-center'],
+  operator: ['home', 'inventory', 'transactions', 'customers', 'orders', 'repairs', 'audit-center'],
+  user: ['home', 'inventory', 'transactions', 'suppliers', 'orders', 'repairs', 'audit-center'],
```

Comment:
You could add a local constant array with the 'lowest common denominators/items'.
['home', 'inventory', 'transactions', 'suppliers', 'customers', 'orders', 'repairs']
Add 'audit-center' to both manager, and operator.
Admin should have all, including 'accounts-manager'.